beets/mediafile.py

```diff
@@ -2005,6 +2005,15 @@ def update(self, dict):
         out_type=float,
     )
 
+    # EBU R128 fields.
+    r128_track_gain = MediaField(
+        StorageStyle(
+            u'R128_TRACK_GAIN',
+            float_places=0
+        ),
```

Comment:
With just a `StorageStyle`, this will only support "free-form" tag formats. On MP3, MPEG-4, and WMA, it will crash unceremoniously. I think the right thing to do is to fill in logical names for these other formats, even if support isn't widespread yet.